### ISSUE TITLE
Fail bootstrapping Salt with Vagrant 1.7.4 

### ISSUE THREAD

**Issue by jarcos** [humanmade/Salty-WordPress#118]
I get this error trying to up a container.
`curl: (23) Failed writing body (630 != 16384)` 

I don't know really the reason why, but after a research I found this on Vagrant repository that solves the problem.
https://github.com/mitchellh/vagrant/issues/6029

**Comment by scottbasgaard** [comment]
@joehoyle just hit this too on upgrading to latest Vagrant / Virtualbox. Might be worth looking into.

**Comment by scottbasgaard** [comment]
Like @jarcos said fix here is replacing the following in your vagrantfile:

```
  config.vm.provision :salt do |salt|
    salt.verbose = true
    salt.minion_config = 'config/salt/minions/vagrant.conf'
    salt.run_highstate = true
  end
```

with:

```
  config.vm.provision :salt do |salt|
    salt.verbose = true
    salt.bootstrap_options= "-F -c /tmp -P"
    salt.minion_config = 'config/salt/minions/vagrant.conf'
    salt.run_highstate = true
  end
```

**Comment by joehoyle** [comment]
Fixed in 2c2ebab, thanks!
